Code/Framework/AzCore/AzCore/Math/Internal/SimdMathVec3_scalar.inl

```diff
@@ -608,10 +608,12 @@ namespace AZ
         }
 
 
+AZ_PUSH_DISABLE_WARNING(4723, "-Wunknown-warning-option") // potential divide by 0 warning
```

Comment:
This warning disabling can be tighten up to be around the return statement instead of the entire function.

Reply:
I tried that first. It only works around the function. There is another case in the case and it goes to say `(needs to wrap the function)`.